test/library/standard/Dataframes/psahabu/AccessSeries.chpl

```diff
@@ -1,9 +1,9 @@
 use Dataframes;
 
-var I = ["A", "B", "C", "D", "E"];
 var A = ["a", "b", "c", "d", "e"];
+var I = ["A", "B", "C", "D", "E"];
 
-var letters: TypedSeries(string) = new TypedSeries(I, A);
+var letters: TypedSeries(string) = new TypedSeries(A, new TypedIndex(I));
```

Comment:
Is the 'new TypedIndex' part temporary?

Reply:
As I was writing the pandas examples, I realized that it was kind of weird to pass bare arrays into the TypedSeries constructor for the index, so now I have the initializer take an Index instead. Hence `new TypedIndex()` is the Index counterpart to `new TypedSeries()` for Series.